### DISCUSSION TITLE
workspace.json file is getting too big

### DISCUSSION THREAD

**Discussion by haroonsiddiqi** [structurizr/java#266] [Q&A]
### Question

workspace.json file is getting too big after I added existing design diagrams (.png). It looks like it is embedding all the images instead of local directory link

![image](https://github.com/structurizr/ui/assets/5772437/528899c4-7d54-4f74-a7db-4385baea0abd)
![image](https://github.com/structurizr/ui/assets/5772437/a542b7f3-0810-4ae9-beba-b05a12ef9f75)

**Comment by simonbrowndotje** [comment]
Image views can be defined with a URL, rather than a local image reference, which will prevent the image from being inlined into the workspace.

See https://github.com/structurizr/examples/blob/main/dsl/big-bank-plc/workspace.dsl#L182 for an example.

**Reply by haroonsiddiqi** [reply]
Thanks Simon, Could you please guide if/how can we host images on the same web server Structurizr is running on and use the URL form to link them. Would really appreciate your guidance on best practice to use existing drawio architecture/design artifacts.
